features/update_membership_application.feature

```diff
@@ -0,0 +1,24 @@
+Feature: As an Admin
+  In order to get members into SHF and get their money
+  I need to be able to accept/reject their application
+  PT: https://www.pivotaltracker.com/story/show/133950603
+
+  Background:
+    Given the following users exists
+      | email                  | admin |
+      | applicant_1@random.com |       |
+      | din@mail.se            |       |
+      | admin@sgf.com          | true  |
```

Comment:
The only users used in this test are 
```
 | din@mail.se            |       |
 | admin@sgf.com     | true  |
````
No need to keep `applicant_1@random.com` around